src/main/pageobjects/amz.footer.page.ts

```diff
@@ -0,0 +1,48 @@
+import Page from "./page";
+
+import * as config from "config"
+const baseURL = config.get('Environment.baseUrl');
+
+export default class Footer extends Page{
+   public async path(link:any)
+    {
+        return browser.url(link)
```

Comment:
why are you explicitly calling browser.url here? page class already has this logic, please use it

Reply:
Done